cli/script/command-executor.ts

```diff
@@ -18,10 +18,11 @@ import wordwrap = require("wordwrap");
 
 import * as cli from "../definitions/cli";
 import { AcquisitionStatus } from "code-push/script/acquisition-sdk";
-import { AccessKey, AccountManager, App, Deployment, DeploymentKey, DeploymentMetrics, Package, UpdateMetrics } from "code-push";
+import { AccessKey, AccountManager, App, Deployment, Collaborator, DeploymentKey, DeploymentMetrics, Package, UpdateMetrics } from "code-push";
 var packageJson = require("../package.json");
 import Promise = Q.Promise;
 var progress = require("progress");
+var emailValidator = require("email-validator");
```

Comment:
Did you intend to actually use this anywhere? I didn't see any local validation of the email fields to ensure they are valid e-mail addresses.